ci/pipeline.yml

```diff
@@ -1107,6 +1109,22 @@ jobs:
             options:
               credentials_source: static
               json_key: '((bosh_release_blobs_gcp_credentials_json))'
+    - task: bump-bosh-blobstore-azure-storage
+      file: bosh-src/ci/tasks/bump-blobstore-cli.yml
```

Comment:
I noticed that `bosh-src/ci/tasks/bump-blobstore-cli.sh` is checking the blobstore type.

The configured `BLOBSTORE_TYPE: azure-storage` would result in `"Error: unknown BLOBSTORE_TYPE='${BLOBSTORE_TYPE}'; exiting"`

Don't we need to update `bosh-src/ci/tasks/bump-blobstore-cli.sh` first?